### ISSUE TITLE
Sort message identifiers

### ISSUE THREAD

**Issue by stwalkerster** [enwikipedia-acc/waca#28]
Currently, message identifiers are based on an auto-increment field which is hard-coded into many places in the code.

This is likely to break if multiple messages are added in parallel, and coded to use the same ID, or different IDs get assigned in the production database.

I propose using an indexed short string identifier to handle this, which can then be hard-coded in such a way that internal database IDs don't matter.

This sort-of blocks #27

**Comment by stwalkerster** [comment]
Won't fix.

The entire architecture of this has changed in newinternal
